agent/functional_tests/README.md

```diff
@@ -59,18 +59,18 @@ done first:
   `iptables -t nat -A OUTPUT -d 169.254.170.2 -p tcp -m tcp --dport 80 -j REDIRECT --to-ports 51679`
   and `export TEST_TASK_IAM_ROLE_NET_HOST=true`
 
-### Windows
+### Custom Agent
 
-Before running these tests, you should build the ECS agent (as `agent.exe`) and
-record the directory where the binary is present in the `ECS_WINDOWS_TEST_DIR`
-environment variable.
+If you want to change the tests to point to a custom agent exe:
```

Comment:
Please retain the other instructions as well.  You can either build or download an agent binary.